dbt/models/reporting/schema.yml

```diff
@@ -235,6 +235,24 @@ models:
               (year >= '2021' OR (
               year = '2020' AND stage_name = 'BOR CERTIFIED')
               )
+      - accepted_range:
+          name: reporting_vw_pin_value_long_exempt_parcels_values
+          column_name: tot
+          config:
+            tags:
+              - test_qc_iasworld
+          max_value: 1
+          where: |
+            class = 'EX' AND CAST(year AS int) = {{ var('test_qc_year_start') }}
```

Comment:
**[Nitpick, non-blocking]** I think this is reasonable and will probably work, but we should probably align it with our other test filters by setting a maximum for the year:

https://github.com/ccao-data/data-architecture/blob/4b1b86885058fc2fae5e623afe82b2f6ed856cf4/dbt/models/iasworld/schema/iasworld.oby.yml#L40

```suggestion
          where: |
            class = 'EX' AND CAST(year AS int) BETWEEN {{ var('test_qc_year_start') }} AND {{ var('test_qc_year_end') }}
```

I honestly can't quite remember why we designed it this way... perhaps there are rows that have a taxyr incorrectly set to the future, or we're being defensive in case a rollover happens before January 1st? Dan will have more context!

Reply:
i'll reopen this if we decide the time window needs to be adjusted. my thought was that reporting on data that can't be fixed would just be a distraction.